test/build/config-optimize/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "This test ensures that buildOptions.optimize: false disables terser",
+  "scripts": {
+    "prepare": "node ../../../pkg/dist-node/index.bin.js",
+    "TEST": "node ../../../pkg/dist-node/index.bin.js build"
+  },
+  "snowpack": {
+    "buildOptions": {
+      "optimize": false
+    },
+    "scripts": {
+      "mount:src": "mount src --to /_dist_"
+    }
+  },
+  "dependencies": {
+    "array-flatten": "^3.0.0"
+  }
+}
```

Comment:
This is a new test that checks to make sure `buildOptions.optimize: false` works